forge-gui/res/editions/Edge of Eternities.txt

```diff
@@ -4,6 +4,54 @@ Date=2025-08-01
 Name=Edge of Eternities
 Type=Expansion
 ScryfallCode=EOE
+TokensCode=TEOE
```

Comment:
I don't think this is needed? I think we look for T by default @Hanmac

Reply:
removed.